drivers/Aqara/aqara-lock/config.yml

```diff
@@ -0,0 +1,6 @@
+name: 'Aqara Lock'
+packageKey: 'aqara-lock'
+permissions:
```

Comment:
This will require the aqara specific permission to give it access to the cloud APIs for the key retrieval.  @tandres can verify but I think it should be 

`aqara-provisioning: {}`

Reply:
![045d9470-05ac-44ad-96c7-5232b69b336e](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/assets/34559415/880561b1-6177-4a6c-be16-07e0091f3e02)

It seems that modifying the provisioning in already deployed driver is not allowed.

After delete the driver in private channel, I can get below error when i type `smartthings edge:drivers:package -p acceptance`
```
14:44 $ smartthings edge:drivers:package -p acceptance
    AxiosError: Request failed with status code 500: {"requestId":"1758721324210160703","error":{"code":"UnexpectedError","message":"A non-recoverable error condition occurred.","details":[]}}
    Code: ERR_BAD_RESPONSE
```
```
timestamp=05:44:59.415, level=ERROR, loggingId=1758721324210160703, logger=s.s.services.PermissionService, userUuid=, msg=, key=forbidden_internal_permission, non system user attempting to upload a driver with internal only permission: permission key=aqara-provisioning, correlationId=1758721324210160703,, 
```
```
timestamp=05:44:59.415, level=ERROR, loggingId=1758721324210160703, logger=s.m.e.h.UnhandledExceptionHandler, userUuid=, msg=, key=api_exception_unhandled,, 
smartthings.common.exceptions.ForbiddenException: Insufficient permission to upload internal only driver
	at smartthings.sprocket.services.PermissionService.validatePackagePermissions(PermissionService.kt:43)
	at smartthings.sprocket.services.DriverPackageService.newDriver(DriverPackageService.kt:179)
	at smartthings.sprocket.services.DriverPackageService.access$newDriver(DriverPackageService.kt:32)
	at smartthings.sprocket.services.DriverPackageService$newDriver$1.invokeSuspend(DriverPackageService.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
```

@varzac Can't we make private channel with `aqara-provisioning: {}`? or is there other way to make private channel? (or need to get system authorization?

Reply:
No, you cannot use that permission in a private channel.  As previously discussed to restrict access to the cloud services to only the official integration, only drivers published by SmartThings can use this permission.  Unfortunately changing permissions is also something we don't allow on a driver once it is published to protect users from a driver publisher changing permissions after they had given it access.  I think the result is that the easiest way to move forward is to address the changes in this PR.  Then open a new PR which will create a new channel, and publish the driver with the new permissions.  We will make sure to give the updated PR immediate attention and get it started through the release process ASAP to prepare for the WWST testing.

Reply:
I forked this branch and opened up a PR to be sure the permission would be present. It looks like the `aqara-provisioning` permission will need to be removed from the feeder driver otherwise CI will fail: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/actions/runs/8603874779